PayPal/src/main/java/com/braintreepayments/api/paypal/PayPalBillingPricing.kt

```diff
@@ -0,0 +1,39 @@
+package com.braintreepayments.api.paypal
+
+import android.os.Parcelable
+import kotlinx.parcelize.Parcelize
+import org.json.JSONObject
+
+/**
+ * PayPal Recurring Billing Agreement pricing details.
+ *
+ * @property pricingModel The pricing model associated with the billing agreement.
+ * @property amount Price. The amount to charge for the subscription, recurring, UCOF or installments.
+ * @property reloadThresholdAmount The reload trigger threshold condition amount when the customer is charged.
+ */
+@Parcelize
+data class PayPalBillingPricing(
+    var pricingModel: PayPalPricingModel,
+    var amount: String,
+    var reloadThresholdAmount: String?
+) : Parcelable {
+
+    constructor(
+        pricingModel: PayPalPricingModel,
+        amount: String
+    ) : this(pricingModel = pricingModel, amount = amount, reloadThresholdAmount = null)
```

Comment:
Same constructor question here. Could we combine this with the main constructor?

Reply:
Done